bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
But in the same time, I'm not sure it needs to be changed.
The advantage of `npmRunSpawn` over `yarn jest` beeing: 
- It does not require yarn to be globally installed to work
- It spawn only one child process, unlike yarn, who will spawn one for itself and one for the command

@christophehurpeau ?